<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatsWindow</class>
 <widget class="QMainWindow" name="ChatsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="chats_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
background-color: rgb(40, 39, 46);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="main_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignTop">
       <widget class="QPushButton" name="menu_push_button">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	border: none;
	background-color: rgb(40, 39, 46);
	color: rgb(40, 39, 45);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../resources/menu-btn.svg</normaloff>../../resources/menu-btn.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>0</number>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="chats_layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QListWidget" name="list_of_chats">
          <property name="styleSheet">
           <string notr="true">QListWidget
{
	background-color: rgb(66, 65, 75)
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="curr_and_msg_layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QListWidget" name="current_chat"/>
        </item>
        <item>
         <widget class="QLineEdit" name="message_input">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit
{
	background-color: rgb(56, 55, 66)
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="settings_button">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="settings_button"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
